packages/packages/commonwell-cert-runner/src/patient-management.ts

```diff
@@ -65,7 +65,7 @@ export async function patientManagement(
   // Main Account Link
   const person = await commonWell.enrollPerson(queryMeta, personStrongId);
   const personId = getId(person);
-  await commonWell.addPatientLink(queryMeta, personId, referenceLink);
+  await commonWell.patientLink(queryMeta, personId, referenceLink);
```

Comment:
I see you pushed a commit with changes, not sure if you're planning to do these on another commit, but just making sure you saw these, re: `patientLink()` was deprecated and calls have been updated to `addPatientLink()`, which this is reverting

Reply:
The merging didnt seem to go well but i reverted it back to addPatientLink